bundles/org.openhab.binding.kostalinverter/src/main/java/org/openhab/binding/kostalinverter/internal/secondgeneration/SecondGenerationHandler.java

```diff
@@ -92,14 +92,24 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 case SecondGenerationBindingConstants.CHANNEL_BATTERYUSAGECONSUMPTIONSET:
                     valueConfiguration = command.toString();
                     dxsEntriesConf = "33556249";
-                    preSetExecuteConfigurationChanges(httpClient, url, username, password, dxsEntriesConf,
-                            valueConfiguration);
+                    try {
+                        preSetExecuteConfigurationChanges(httpClient, url, username, password, dxsEntriesConf,
+                                valueConfiguration);
+                    } catch (Exception handleCommandException) {
+                        logger.debug("Handle command for {} on channel '{}: {}'", thing.getUID(), httpClient,
+                                handleCommandException.getMessage());
+                    }
```

Comment:
Please move your try before your switch so that you will have only one try/catch. This makes sense especially when all your catch code are similar.